doc/source/conf.py

```diff
@@ -124,9 +124,9 @@
 # Special mocking of packaging.version.Version is required when using sphinx;
 # we can't just add this to autodoc_mock_imports, as packaging is imported by
 # sphinx even before it can be mocked. Instead, we patch it here.
-import packaging
+import packaging.version as packaging_version
```

Comment:
I found this is broken when using Python > 3.8 because you cannot just access packaging.verison

Reply:
I guess packaging version could also matter. I think depending on how you import "version", you cannot access it via packaging.version, but only via from package import version